src/main/scala/fs2/Chunk.scala

```diff
@@ -29,6 +29,16 @@ trait Chunk[+A] { self =>
     iterator.map(f).copyToBuffer(buf)
     Chunk.indexedSeq(buf)
   }
+  /**
+   * `[[scanLeft]]` is a helper for process1.scan; it differs
+   * from standard collection behavior by not including
+   * the initial value in the output.
```

Comment:
Dropping the value here is so that we don't repeat values across chunk boundaries; the first value being emitted is special-cased in the process1 implementations.

I checked this already, but the details are fiddly, so I'll double-check in a while and make sure that they match.